characterprofile.php

```diff
@@ -1,22 +1,40 @@
-<?php require_once 'engine/init.php'; include 'layout/overall/header.php'; 
-if ($config['log_ip']) {
+<?php require_once 'engine/init.php'; include 'layout/overall/header.php';
+ 
+if ($config['log_ip']) 
+{
 	znote_visitor_insert_detailed_data(4);
 }
-if (isset($_GET['name']) === true && empty($_GET['name']) === false) {
+
+if (isset($_GET['name']) === true && empty($_GET['name']) === false) 
+{
 	$name = $_GET['name'];
 	$user_id = user_character_exist($name);
-	if ($user_id !== false) {
-		if ($config['TFSVersion'] == 'TFS_10') {
-			$profile_data = user_character_data($user_id, 'name', 'level', 'vocation', 'lastlogin', 'sex');
+	
+	if ($user_id !== false) 
+	{	
+		if ($config['TFSVersion'] == 'TFS_10') 
+		{
+			$profile_data = user_character_data($user_id, 'account_id', 'name', 'level', 'vocation', 'health', 'healthmax', 'experience', 'mana', 'manamax', 'sex', 'lastlogin');
 			$profile_data['online'] = user_is_online_10($user_id);
-			if ($config['Ach'] == true) {
+			
+			if ($config['Ach']) 
+			{
 				$achievementPoints = mysql_select_single("SELECT SUM(`value`) AS `sum` FROM `player_storage` WHERE `key` LIKE '30___' AND `player_id`='$user_id'");
```

Comment:
```player_id`='$user_id'````
Player id is an integer, so cast it to int (or make sure it is an int) before using it in a query.